liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/jsp/CompileJspMojo.java

```diff
@@ -17,24 +17,24 @@
 import org.codehaus.plexus.util.xml.Xpp3Dom;
 
 import io.openliberty.tools.ant.jsp.CompileJSPs;
-import io.openliberty.tools.maven.BasicSupport;
+import io.openliberty.tools.maven.InstallFeatureSupport;
```

Comment:
This file is also missing a copyright. Figure out when it was created in order to set the correct years in it.